magiceightball/src/App.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useState } from "react";
+import "./styles.css";
+import { FaMagic } from 'react-icons/fa';
+
+
+
+function App() {
+    const [question, setQuestion] = useState("");
+    const [answer, setAnswer] = useState("");
+    const [errorMessage, setErrorMeessage] = useState("");
+    const [isLoading, setIsLoading] = useState(false);
+    const [imageSrc, setImageSrc] =useState("");
+
+    const handleChange = (e) => {
+      setQuestion(e.target.value);
+    }
+    
+    const handleSubmit = (e) => {
+      e.preventDefault();
+      if(question.slice(-1) !== '?' || question === ""){
+        setErrorMeessage("You didn't ask a question!");
+        setIsLoading(false);
+        return;
+      } else {
+        setErrorMeessage("");
+        setIsLoading(true);
+        setAnswer("");
+        fetch("https://yesno.wtf/api")
```

Comment:
try to use async/await (this is also OK but async await is more common now) It'll be better to learn async/await, too